config/develop/namespaced/glue-job-run-great-expectations-on-parquet.yaml

```diff
@@ -0,0 +1,18 @@
+template:
+  path: glue-job-run-great-expectations-on-parquet.j2
+dependencies:
+  - develop/glue-job-role.yaml
+stack_name: "{{ stack_group_config.namespace }}-glue-job-RunGreatExpectationsParquet"
+parameters:
+  Namespace: {{ stack_group_config.namespace }}
+  JobDescription: Runs great expectations on a set of data
+  JobRole: !stack_output_external glue-job-role::RoleArn
+  TempS3Bucket: {{ stack_group_config.processed_data_bucket_name }}
+  S3ScriptBucket: {{ stack_group_config.template_bucket_name }}
+  S3ScriptKey: '{{ stack_group_config.namespace }}/src/glue/jobs/run_great_expectations_on_parquet.py'
+  GlueVersion: "{{ stack_group_config.json_to_parquet_glue_version }}"
```

Comment:
It's not necessarily the case that JSON to Parquet and this GX job will always use the same Glue version. My recommendation is to hardcode the version here and in `config/prod` since it's unlikely to be referenced elsewhere, but you could add an additional variable to the config file if you prefer.

Reply:
Do you envision other GX jobs to use all the same Glue version (if they use glue)?

Reply:
Yes.